src/connections/ProjectPitch.ts

```diff
@@ -0,0 +1,133 @@
+  uuid: string;
+};
+
+const projectPitchConnection: Connection<ProjectPitchConnection, ProjectPitchConnectionProps> = {
+  load: ({ projectPitch }, { uuid }) => {
+    if (projectPitch == null) projectPitchGet({ pathParams: { uuid } });
+  },
+
+  isLoaded: ({ projectPitch }) => projectPitch !== null,
+  selector: selectorCache(
+    ({ uuid }: ProjectPitchConnectionProps) => uuid,
+    ({ uuid }: ProjectPitchConnectionProps) =>
+      createSelector(
+        [
+          projectPitchGetIsFetching({ pathParams: { uuid } }),
+          projectPitchGetFetchFailed({ pathParams: { uuid } }),
+          projectPitchesSelector
+        ],
+        (isLoading, requestFailed, selector) => ({
+          isLoading,
+          requestFailed,
+          projectPitch: selector[uuid]?.attributes ?? null
+        })
+      )
+  )
+};
+
+export type ProjectsPitchesConnection = {
+  fetchFailure: PendingErrorState | null;
+  data?: ProjectPitchDto[];
+  indexTotal?: number;
+  refetch: () => void;
+};
+
+type ProjectPitchIndexFilterKey = keyof Omit<
+  ProjectPitchIndexQueryParams,
+  "page[size]" | "page[number]" | "sort[field]" | "sort[direction]"
+>;
+
+export type ProjectPitchIndexConnectionProps = {
+  pageSize?: number;
+  pageNumber?: number;
+  sortField?: string;
+  sortDirection?: "ASC" | "DESC";
+  filter?: Partial<Record<ProjectPitchIndexFilterKey, string>>;
+};
+
+const projectPitchIndexQuery = (props?: ProjectPitchIndexConnectionProps) => {
+  const queryParams = {
+    "page[number]": props?.pageNumber,
+    "page[size]": props?.pageSize
+  } as ProjectPitchIndexQueryParams;
+  if (props?.sortField != null) {
+    queryParams["sort[field]"] = props.sortField;
+    queryParams["sort[direction]"] = props.sortDirection ?? "ASC";
+  }
+  if (props?.filter != null) {
+    for (const [key, value] of Object.entries(props.filter)) {
+      (queryParams as Record<string, string | number | undefined>)[key] = value;
+    }
+  }
+  return queryParams;
+};
+
+const indexCacheKey = (props: ProjectPitchIndexConnectionProps) => getStableQuery(projectPitchIndexQuery(props));
+
+const projectPitchesIndexParams = (props?: ProjectPitchIndexConnectionProps) => ({
+  queryParams: projectPitchIndexQuery(props)
+});
+
+const projectPitchesConnection: Connection<ProjectsPitchesConnection, ProjectPitchIndexConnectionProps> = {
+  load: ({ data }, props) => {
+    if (!data) projectPitchIndex(projectPitchesIndexParams(props));
+  },
+
+  isLoaded: ({ data }) => data !== undefined,
+  selector: selectorCache(
+    props => indexCacheKey(props),
+    props =>
+      createSelector(
+        [
+          projectPitchIndexIndexMeta("projectPitches", projectPitchesIndexParams(props)),
+          projectPitchIndexFetchFailed(projectPitchesIndexParams(props)),
+          projectPitchesSelector
+        ],
+        (indexMeta, fetchFailure, selector) => {
+          console.log(indexMeta);
+          const refetch = () => ApiSlice.pruneIndex("projectPitches", "");
```

Comment:
Is anything using this `refetch` method? I'm not finding any users in this PR, and I can't think of why it would be needed for this connection.